src/aat/resources/features/F-048/F-048.feature

```diff
@@ -0,0 +1,63 @@
+@F-048
+Feature: Get the pagination metadata for a case data search for Case Worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-136
+  Scenario: must return pagination metadata successfully for correct inputs
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    #And the request [is prepared with valid Jurisdiction, Case ID and User ID]
+    And it is submitted to call the [Get the pagination metadata for a case data search for Case Worker] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [returns the pagination metadata]
+    And the response has all the details as expected
+
+  @S-133
+  Scenario: must return 400 if the sort direction in input parameters is not in ASC or DESC
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    #And the request [contains invalid Sort Direction]
```

Comment:
[contains invalid Sort Direction] --> [contains an invalid Sort Direction]

Let's use as correct a grammar as possible.

Reply:
Because we want the scenarios to read as smooth as possible.

Reply:
Agreed and updated.